src/styles/blocs/card.scss

```diff
@@ -0,0 +1,112 @@
+.catalog__card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #F3F3F3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding-inline: 16px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+
+  &-img {
+    height: 134px;
+    width: 160px;
+    display: block;
+    margin: 0 auto 40px;
+    flex-shrink: 0;
+  }
+
+  &-title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color:#060B35;
+    margin: 0;
+    margin-bottom: 4px;
+  }
+
+  &-product-code {
+    margin: 0;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary;
+    margin-bottom: 16px;
+  }
+
+  &-stars-and-reviews {
+    display: flex;
+    margin: 0;
+    margin-bottom: 24px;
+    justify-content: space-between;
+  }
+
+  &-reviews {
+    font-weight: 400;
+    height: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    color:$main-color;
+    display: flex;
+    justify-content: end;
+    align-items: flex-end;
+  }
+
+  &-price {
+    margin: 0;
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+
+    &-name {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $secondary;
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $main-color;
+      display: flex;
+      justify-content: right;
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
+    transition: all 300ms ease;
```

Comment:
it's not a good practice to use `all`, specify what properties you'd like to change exactly